.eslintignore

```diff
@@ -0,0 +1,2 @@
+**/node_modules/**
```

Comment:
You want to include all stuff that is in gitignore as well. I started to like putting a message to keep them in sync like done here:
https://github.com/api3dao/airseeker/pull/1/files#diff-a0fdacd2ade87c5238dde44378f574f7e123e669acdf8b740878b14b33b20275R1

Reply:
Thanks for this!

Reply:
I've updated all the ignore files :)